props/prop_base.h

```diff
@@ -866,6 +878,23 @@ class PropBase : CommandParser, Looper, protected SaberBase {
   uint32_t last_on_time_;
 #endif
 
+#ifdef ENABLE_MENU_DIMBLADE
+  bool change_brightness_ = false;
```

Comment:
Why do we need to have this both here and in SaberBase?

Reply:
You are correct. Made the adjsutment to use SaberBase::GetDimChangeMode()